lib/contact_sync.rb

```diff
@@ -1,13 +1,16 @@
+require 'pry-rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.